src/scene/materials/lit-options.js

```diff
@@ -0,0 +1,102 @@
+import {
+    BLEND_NONE, GAMMA_NONE
+} from '../constants.js';
+
+class LitOptions {
```

Comment:
we should add the documentation to some of those properties that is still stored in the old place .. see my other comment on this PR.

Reply:
Do you mean the properties stored in the ordinary options object?